lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -854,6 +900,12 @@ def _rel_data_redacted_dict(
                 "access-key": self.secrets.get_secret_id(Scope.APP, "s3-access-key"),
                 "secret-key": self.secrets.get_secret_id(Scope.APP, "s3-secret-key"),
             }
+
+        if rel_data.credentials and getattr(rel_data.credentials.s3, "s3_tls_ca_chain", None):
+            sid = self.secrets.get_secret_id(Scope.APP, "s3-tls-ca-chain")
+            if sid:
```

Comment:
```suggestion
            if sid := self.secrets.get_secret_id(Scope.APP, "s3-tls-ca-chain"):
```

Can you also rename this variable?

Reply:
Done